### ISSUE TITLE
wysiwyg bugs

### ISSUE THREAD

**Issue by ragesoss** [WikiEducationFoundation/WikiEduDashboard#606]
These may be trix bugs (and we're not on the latest release). Keeping track of what people report going wrong, in any case...
- "Sometimes when I’m editing a box, all the bold formatting mysteriously disappears."
- "When I try to indent some of the bulleted lists [...] nothing happens. Sometimes it works one day but not another."
- The current text in the Trix input field is not accessible via screen reader.

**Comment by thenickcox** [comment]
Any browser/OS data on these?

**Comment by ragesoss** [comment]
Looks like the reporting user was on Chrome 47, OS X.

**Comment by thenickcox** [comment]
Yep. 

Steps to repro: 

[course_slug]/timeline -> edit block, add bold -> save -> click edit again

**Comment by ragesoss** [comment]
I'm looking into changing WYSIWYG editors, mainly because we have a significant need to retain CSS class attributes in Timeline blocks, and Trix removes them upon save.

The most promising so far is TinyMCE, which is the one used in WordPress and could allow switching back and forth between WYSIWYG and html modes.

**Comment by ragesoss** [comment]
I deployed TinyMCE today, which I think ought to be much more screen-reader friendly, as they have been around for a lot longer and claim to have implemented a11y standards.

I'll close this, and open a new issue if there are a11y issues with the new editor.
